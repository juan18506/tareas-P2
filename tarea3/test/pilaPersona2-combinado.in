crearPila

imprimirCantidadPila

# Agrego a Persona 5 a la Pila
crearPersona 5 34 pedro
apilar
liberarPersona

imprimirCantidadPila

# Agrego a Persona 2 a la Pila
crearPersona 2 12 ana
apilar
liberarPersona

imprimirCantidadPila

# Agrego a Persona 9 a la Pila
crearPersona 9 26 pepe
apilar
liberarPersona

imprimirCantidadPila

obtenerCimaPila

# Elimino a Persona 9 de la Pila
desapilar

imprimirCantidadPila

obtenerCimaPila

# Elimino a Persona 2 de la Pila
desapilar

imprimirCantidadPila

obtenerCimaPila

# Elimino a Persona 5 de la Pila
desapilar


liberarPila

Fin