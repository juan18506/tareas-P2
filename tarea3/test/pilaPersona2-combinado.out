1>2>La cantidad de personas en la pila es: 0.
3>#  Agrego a Persona 5 a la Pila.
4>5>6>7>La cantidad de personas en la pila es: 1.
8>#  Agrego a Persona 2 a la Pila.
9>10>11>12>La cantidad de personas en la pila es: 2.
13>#  Agrego a Persona 9 a la Pila.
14>15>16>17>La cantidad de personas en la pila es: 3.
18>Persona 9:  pepe, 26 años
19>#  Elimino a Persona 9 de la Pila.
20>21>La cantidad de personas en la pila es: 2.
22>Persona 2:  ana, 12 años
23>#  Elimino a Persona 2 de la Pila.
24>25>La cantidad de personas en la pila es: 1.
26>Persona 5:  pedro, 34 años
27>#  Elimino a Persona 5 de la Pila.
28>29>30>Fin.
